tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/GenerateContent.java

```diff
@@ -115,6 +115,7 @@ public void execute() throws BaseNessieClientServerException {
 
     ThreadLocalRandom random = ThreadLocalRandom.current();
 
+    @SuppressWarnings("JavaTimeDefaultTimeZone")
     String runStartTime =
         DateTimeFormatter.ofPattern("yyyy-MM-dd-HH-mm-ss").format(LocalDateTime.now());
```

Comment:
should we prefer `LocalDateTime.now(ZoneOffset.UTC);` over the suppression?

Reply:
Maybe not. It's a "user" visitble thing